app/src/main/java/org/cru/godtools/GodToolsApplication.java

```diff
@@ -75,4 +76,12 @@ protected void startServices() {
                 .clientId(THEKEY_CLIENTID)
                 .eventsManager(new EventBusEventsManager());
     }
+
+    @Override
+    protected void configureAnalyticsServices() {
+        super.configureAnalyticsServices();
+
+        // Need to keep AppSee in Gradle file in order to work
+        AppSeeAnalyticService.getInstance(getApplicationContext());
```

Comment:
you can just pass in `this` instead of needing to call `getApplicationContext()`